tests/components/basics.test.ts

```diff
@@ -1006,4 +1006,36 @@ describe("t-out in components", () => {
       "<div>&lt;b&gt;one&lt;/b&gt;<b>one</b>&lt;b&gt;two&lt;/b&gt;<b>two</b>&lt;b&gt;tree&lt;/b&gt;<b>tree</b></div>"
     );
   });
+
+  test("can mount a component with no text", async () => {
```

Comment:
title does not seem to match the test? maybe it could be "can update t-out content"